pyquil/api/qvm.py

```diff
@@ -58,6 +64,18 @@ def __init__(self, sync_endpoint='https://api.rigetti.com', async_endpoint='http
         :param random_seed: A seed for the QVM's random number generators. Either None (for an
                             automatically generated seed) or a non-negative integer.
         """
+        # Assert that it is not the case that both device.noise_model is supplied, and gate_/measurement_noise.
```

Comment:
I don't think this comment does/should add much, the code should speak for itself.